### PR TITLE
Add namespace viewer and automatic bucket allocator

### LINKED ISSUE #410: Add automatic namespace allocation to recipe editing page
When creating a recipe, instead of specifying a bucket start value, operators should be able to choose automatic. Then NDT will use a method similar to the existing [namespace tool](https://normandy-namespaces.web.app/) to choose an available bucket in the selected namespace, if one is available.

### PR DESCRIPTION
**Pull request by mythmon**
Fixes #410

## Namespace page
image

## Auto start value
image

### CODE REVIEW

File: content/components/pages/NamespacesPage.tsx [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+        )}
+      {namespacesInfo.isLoading("updateNamespace", selectedNamespace) ? (
+        <div className="grid-layout grid-3 card-grid mt-2">
+          <CardPlaceholder />
+          <CardPlaceholder />
+          <CardPlaceholder />
+        </div>
+      ) : (
+        <div className="grid-layout grid-3 card-grid mt-2">
+          {recipeLists.ok.map(({ recipe, bucketRange }) => (
```
Comment (rehandalal) [inline content/components/pages/NamespacesPage.tsx]: i find these long ternary situations kind of hard to read. i always prefer something like:

```js
let something = <div>stuff here</div>;
if (condition) {
  something = <div>other stuff</div>;
}

...

{something}
```

File: package.json [hunk 2]
```diff
@@ -43,6 +43,7 @@
     "@testing-library/react": "11.2.2",
     "@testing-library/user-event": "12.2.2",
     "@types/auth0-js": "9.14.2",
+    "@types/classnames": "^2.2.11",
```
Comment (rehandalal) [inline package.json]: nit: pin

**Review by rehandalal — CHANGES_REQUESTED** [review]
phew! ok so this looks good for the most part. nice work!! just a couple little things

File: package.json (hunk 2 above)
Comment (mythmon) [reply package.json]: Ugh, I thought I added configuration to stop this.

**Review by rehandalal — APPROVED** [review]
thanks!

bors r+

**Comment by mythmon** [comment]
bors r= @rehandalal

**Comment by mythmon** [comment]
bors r= @rehandalal

**Pull request merged**